mkdocs.yml

```diff
@@ -62,6 +62,7 @@ markdown_extensions:
   - attr_list
   - md_in_html
   - admonition
+  - codehilite
```

Comment:
is this needed for the `Examples` section in the docstrings?

Reply:
That's what the docs from material say, but I've checked and we don't really need it apparently, so I'm removing it.